drush/Commands/SiteInstallCommands.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Drush\Commands;
+
+use Consolidation\AnnotatedCommand\CommandData;
+use Consolidation\AnnotatedCommand\CommandResult;
+use Drupal\cohesion\Drush\DX8CommandHelpers;
+
+/**
+ * Rebuild site studio when site is installed via drush.
+ */
+class SiteInstallCommands extends DrushCommands {
+
+  /**
+   * Do a forceful rebuild whenever site is installed.
+   *
+   * @hook post-command site-install
+   */
+  public function postCommand($result, CommandData $commandData) {
+    $this->yell('Finished rebuilding.');
```

Comment:
@saurabhtripathics - let's also add some additional checks here please -

1. check if profile being installed is acquia_cms. We don't want this to trigger for any other profile install
2. check if site studio modules are already installed

Reply:
yes done these changes